components/http_client.rst

```diff
@@ -600,25 +600,40 @@ class to autoconfigure the HTTP client based on the requested URL::
                 'Authorization' => 'token '.$githubToken,
             ],
         ],
+        // ...
     ]);
 
+You can define several scopes, so that each set of options is added only if a
+requested URL matches one of the regular expressions provided as keys.
+
 If the request URL is relative (because you use the ``base_uri`` option), the
 scoping HTTP client can't make a match. That's why you can define a third
 optional argument in its constructor which will be considered the default
 regular expression applied to relative URLs::
 
     // ...
```

Comment:
Please add the use statement for the ScopingHttpClient

Reply:
it's already a few lines above

Reply:
Didn't see it on the phone 👍